freqtrade/main.py

```diff
@@ -47,8 +54,8 @@ def main(sysargv: List[str]) -> None:
     except OperationalException as e:
         logger.error(str(e))
         return_code = 2
-    except BaseException:
-        logger.exception('Fatal exception!')
+    except BaseException as e:
+        logger.exception('Fatal exception! ' + str(e))
```

Comment:
Note that this is not necessary. 
[from the logging logs](https://docs.python.org/3/library/logging.html#logging.Logger.exception)

> Logs a message with level ERROR on this logger. The arguments are interpreted as for debug(). Exception info is added to the logging message. This method should only be called from an exception handler.

Reply:
without this:
```
Fatal exception!
Traceback (most recent call last):
  File "/home/user/freqtrade-wrk/github-hroff-1902/freqtrade/freqtrade/main.py", line 31, in main
    raise SystemError("Freqtrade requires Python version >= 3.6")
SystemError: Freqtrade requires Python version >= 3.6
```
with this:
```
Fatal exception! Freqtrade requires Python version >= 3.6
Traceback (most recent call last):
  File "/home/user/freqtrade-wrk/github-hroff-1902/freqtrade/freqtrade/main.py", line 31, in main
    raise SystemError("Freqtrade requires Python version >= 3.6")
SystemError: Freqtrade requires Python version >= 3.6
```

Hmm, in some of my early tests of this functionality I have not fount the 'Freqtrade requires Python version >= 3.6' string in the printed test, so I added this. Maybe it's redundant, right...